README.md

```diff
@@ -92,9 +92,9 @@ The following section is useful for Dockstore developers (e.g. those that want t
 ### Dependencies
 
 The dependency environment for Dockstore is described by our 
-[Travis-CI config](https://github.com/dockstore/dockstore/blob/develop/.travis.yml). In addition to the dependencies for 
+[CircleCI config](https://github.com/dockstore/dockstore/blob/develop/.circleci/config.yml). In addition to the dependencies for 
 Dockstore users, note the setup instructions for postgres. Specifically, you will need to have postgres installed 
```

Comment:
Depending on how far we want to take this PR, you could mention oue docker-compose.yml as a way to run Postgres (and ES). Comment also applies to line 125 (which I can't comment on, because it wasn't changed).